backend/internal/storage/postgres/schema/review.go

```diff
@@ -92,6 +91,79 @@ func (r *ReviewRepository) GetReviewsByUserID(ctx context.Context, id string) ([
 	return reviews, nil
 }
 
+func (r *ReviewRepository) UpdateReview(ctx context.Context, review *models.Review, updateComment bool, updateRating bool) (*models.Review, error) {
+
+	// Declare these variables first to allow for assignment inside if statement body
+	var query string
+	var err error
+	var updatedReview models.Review
+
+	if updateRating && updateComment {
+		query = `
+        UPDATE review 
+        SET comment = $1, rating = $2, updated_at = $3 
```

Comment:
```suggestion
        SET comment = $1, rating = $2, updated_at = now() 
```
this is postgres' [now()](https://www.postgresqltutorial.com/postgresql-date-functions/postgresql-now/) function for getting the time so that you can avoid passing it around. just make sure to update the id parameter to $3 instead of $4

Reply:
fixed!